scripts/emoji_codegen.py

```diff
@@ -12,10 +12,35 @@ def __init__(self, code, shortname, unicodename):
         self.shortname = shortname
         self.unicodename = unicodename
 
+def generate_provider_class(**kwargs):
+    entrycount = sum([len(c[1]) for c in kwargs.items()])
+    tmpl = Template('''\
+    // SPDX-FileCopyrightText: Nheko Contributors
+    //
+    // SPDX-License-Identifier: GPL-3.0-or-later
+
+    // DO NOT EDIT Provider.h DIRECTLY! EDIT IT IN scripts/emoji_codegen.py AND RUN scripts/codegen.sh!
+
+    #pragma once
+    #include <QObject>
```

Comment:
This include seems unnecessary.

Reply:
I assumed it was needed for the macros. I guess not.